### DISCUSSION TITLE
Grouping of Systems

### DISCUSSION THREAD

**Discussion by stijink** [henrygd/beszel#98] [Ideas]
Hi all,

i would suggest to possibility of grouping systems. The idea arose to me because i have some physical / bare metal hosts as well as virtualized hosts. The grouping of the systems would allow to have all hosts of one group on top of the page and the other / others below that.

**Comment by svenvg93** [comment]
Its on the list, see https://github.com/henrygd/beszel/issues/87 for more information :)

**Reply by stijink** [reply]
@svenvg93 Ah. Great news. Sorry for the duplicate then.

**Reply by svenvg93** [reply]
No worries :)
